PKGBUILD

```diff
@@ -0,0 +1,56 @@
+# Maintainer: Jonas Renk <jonas.renk@stud.tu-darmstadt.de>
+pkgname=systemverilog
```

Comment:
Given that this only is a toolchain based on open-source software and, at least in our context and more importantly, is used for educational purposes only, something like "systemverilog-edu" or "sv-toolchain-edu" seems more appropriate.

Reply:
"sv-toolchain-edu" seems like a good choice to me. Especially since I've named the executables 'sv-*'.